app/move/controllers/assign/confirmation.js

```diff
@@ -0,0 +1,25 @@
+const allocationService = require('../../../../common/services/allocation')
+
+const PersonAssignBase = require('./base')
+
+class ConfirmationController extends PersonAssignBase {
+  middlewareLocals() {
+    super.middlewareLocals()
+    this.use(this.setAssignNext)
+  }
+
+  async setAssignNext(req, res, next) {
+    const allocationId = res.locals.move.allocation.id
+
+    const allocation = await allocationService.getById(allocationId)
```

Comment:
This will need to be wrapped in a `try/catch` in case the promise doesn't resolve.

Reply:
👍